code/go/0chain.net/smartcontract/storagesc/config.go

```diff
@@ -537,13 +538,13 @@ func getConfiguredConfig() (conf *Config, err error) {
 	return
 }
 
-func (ssc *StorageSmartContract) setupConfig(
+func InitConfig(
 	balances chainState.StateContextI) (conf *Config, err error) {
 
 	if conf, err = getConfiguredConfig(); err != nil {
 		return
 	}
-	_, err = balances.InsertTrieNode(scConfigKey(ssc.ID), conf)
+	_, err = balances.InsertTrieNode(STORAGESC_CONFIG_KEY, conf)
```

Comment:
Insert only when MPT does not have the config(retunr util.ErrValueNotPresent error), otherwise the config will be reset to default after restarting